docs/faqs/browser-agent.md

```diff
@@ -0,0 +1,15 @@
+# Injecting Browser Agent
+
+Q: How can I inject the [New Relic Browser Agent](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/) into a Next.js project?
+
+A: It depends on if you are using the Pages or App Router for Next.js. 
+
+
+## Inject Browser Agent with Pages Router
+
+This [Pages Router](https://github.com/newrelic/newrelic-node-examples/blob/main/nextjs-legacy/pages/_document.jsx) example app demonstrates how to inject the browser agent.
```

Comment:
I think the link should be to a specific commit hash. It's possible for a blob/main/ link to become invalid.